src/appleseed/renderer/modeling/light/sunlight.cpp

```diff
@@ -100,6 +100,7 @@ namespace
             m_inputs.declare("turbidity", InputFormatFloat);
             m_inputs.declare("radiance_multiplier", InputFormatFloat, "1.0");
             m_inputs.declare("size_multiplier", InputFormatFloat, "1.0");
+            m_inputs.declare("distance", InputFormatFloat, "1.0");
```

Comment:
`1.0` here is the default value of this input. In appleseed, input default values must usually be defined in two places: in input declarations (here) and in input metadata. It's unfortunate, but until we fix this design issue we must specify default values in these two places.